profile/README.md

```diff
@@ -34,3 +34,11 @@ This requirement sets a bar for transparency, and a foundation for future innova
 
 This section will be developed as the project gets under way. In the mean time, feel free to reach out directly to team members or through our [GitHub
 Discussions](https://github.com/orgs/ccmmf/discussions).
+
+## Licenses and Copyright
+
+A central focus of this project is to generate open source software and open data that will enable transparency while facilitating reuse, collaboration, and derivative works. To this end, we plan to release our works under the following licenses:
+   
+- Software: BSD 3-Clause or compatible license
```

Comment:
This is a good point - the motivation was that some of our code will be released as part of pecan, which hasn't been completely relicensed.

Reply:
Since PEcAn is a separate project (and thus not "our works") with its own existing licensing, I don't think that needs a disclaimer when talking about CCMMF's org-level policies.

Reply:
```suggestion
- Software: BSD 3-Clause 
```